frontend/src/services/httpClient.ts

```diff
@@ -1,6 +1,9 @@
 import axios from "axios";
 import _ from "lodash";
+import {useAppSelector} from "../stores/redux/hooks";
+import {selectLoggedUser} from "../stores/redux/loggedUser/loggedUserSlice";
 
+const loggedUser = useAppSelector(selectLoggedUser);
```

Comment:
This is a correct idea to add token in interceptors. But I'm not sure that hooks work outside of react component. Most likely you need to import directly store and get target state (see next comment)